### ISSUE TITLE
Load backup session toggle in renku-ui

### ISSUE THREAD

**Issue by nperraud** [SwissDataScienceCenter/renku-ui#643]
I believe it would be useful to have a small toggle in renku-ui to load the latest backup instead of the simple information that it will be loaded. 

This is probably important if some people work offline as they may do additional commit and then an automatic option might be very messy.

**Comment by lorenzo-cavazzi** [comment]
The autosaved content is restored based on the commit you selected when you started the interactive environment. If you have unsaved work for commit A and you work offline ending up in commit B then push, when you try to start a new environment from commit B it doesn't restore any unsaved work. This means that case 2 shouldn't be problematic, the worst case is ending up with a lot of autosaved branches.

Anyway, I agree having a toggle/button that allows discarding the autosaved content would be useful.

**Comment by ciyer** [comment]
@nperraud Would instructions on how to clean-up the autosave branch also be an acceptable solution for you?

I think we want to encourage people to look at the autosave branch and either merge the changes or discard them, so I would prefer a solution that is a step towards that goal.

**Comment by rokroskar** [comment]
> The autosaved content is restored based on the commit you selected when you started the interactive environment.

We should make sure this is indeed the case - if @nperraud was having merge conflicts it makes me think that it was _not_ behaving as intended.

**Comment by lorenzo-cavazzi** [comment]
@rokroskar I just verified and there is indeed something wrong. If I start an environment from an old commit, the autosave branch generated when I stop it references the latest commit anyway. Let me check more in depth where is the problem

**Comment by rokroskar** [comment]
Great, thanks for looking into it!

**Comment by lorenzo-cavazzi** [comment]
For info, a fix to that problem has been implemented here SwissDataScienceCenter/renku-notebooks#220

Thanks @nperraud for reporting, I gave it for granted that everything was working correctly but in fact it wasn't

**Comment by nperraud** [comment]
Sorry for my late answer. I wasn't notified of this discution... Weird.

I did not experienced any conflict. However, I found it quite annoying to always have to go gitlab to remove backup branches... @ciyer So for me, the instructions were not the problem. @lorenzo-cavazzi I like the idea of a button, but I understand the point of @ciyer . So what about a simple functionality that allows the user to delete old backup branches that are not at the head?
